ddl/column.go

```diff
@@ -402,7 +373,7 @@ func onAddColumns(d *ddlCtx, t *meta.Meta, job *model.Job) (ver int64, err error
 				}
 			}
 			tblInfo.Columns = append(tblInfo.Columns, newCols[i])
-			adjustColumnInfoInAddColumn(tblInfo, offsets[i])
+			tblInfo.MoveColumnInfo(len(tblInfo.Columns)-1, offsets[i])
```

Comment:
Maybe `onAddColumns` will not reachable after this PR.

Reply:
Nope, the DDL like `alter table t add column a int, b int` will still use `onAddColumns`.

Reply:
It's an illegal stmt in mysql. Shoud we keep this?

Reply:
Seems we support this stmt in parser for a long time. Just keep it right now.